R/seeding.R

```diff
@@ -0,0 +1,48 @@
+##' Seeding function for use with the particle filter
+##' 
+##' @title Compare model to ICU data
+##' 
+##' @param model An \code{odin_model} object
```

Comment:
A bit torn whether `odin_model` (used in models.R and less ambiguous) or just `model` (used in both sampler.R and compare.R) is best. Up to you

Reply:
Gone for `odin_model`